app/config/image_variations.yml

```diff
@@ -20,3 +20,9 @@ ezpublish:
                     reference: ~
                     filters:
                         - { name: geometry/scalewidthdownonly, params: [710] }
+liip_imagine:
+    driver: gd
+    filter_sets:
+        blog_thumbnail:
```

Comment:
Maybe we should define a global parameter for the conversion. And override this default parameter, if we want to.
What do you think?